src/sentry/incidents/endpoints/serializers.py

```diff
@@ -242,22 +244,27 @@ def _handle_actions(self, alert_rule_trigger, actions):
                     )
                 else:
                     action_instance = None
-
                 action_serializer = AlertRuleTriggerActionSerializer(
                     context={
                         "alert_rule": alert_rule_trigger.alert_rule,
                         "trigger": alert_rule_trigger,
                         "organization": self.context["organization"],
                         "access": self.context["access"],
+                        "use_async_lookup": self.context.get("use_async_lookup"),
                     },
                     instance=action_instance,
                     data=action_data,
                 )
 
                 if action_serializer.is_valid():
-                    action_serializer.save()
+                    try:
+                        action_serializer.save()
+                    except ChannelLookupTimeoutError as e:
+                        channel_lookup_timeout_error = e
```

Comment:
just to clarify, we don't raise here because we don't want the timeout error to block saving the rule, we just want to know that we have to kick off the async task, yes?

I'm not sure if we should put a comment in here (and also in the trigger serializer) to make it clear

Reply:
@MeredithAnya comment sounds good